test/controllers/slides_controller_test.rb

```diff
@@ -233,4 +233,34 @@ class SlidesControllerTest < ActionController::TestCase
     assert_select "video[controls]"
     assert_select "video[muted]"
   end
+
+  test "check for proper options for admission labels on scheduled slides" do
+    # Arrange
+    sign_in users(:super_admin)
+    slide = scheduled_items(:standard)
+
+    # Assume
+
+    # Act
+    get :preview, id: slide
+    assert_response :success
+
+    # Assert
+    assert_equal "---\n- Free\n- Tickets Required\n- Free - Registration Required\n- Sold out\n- Cancelled\n", slide.admission
```

Comment:
This test looks brittle. Use assert_css_select here. Also include a comment linking to issue to explain why we're testing something that might seem pretty obvious.

Also, to confirm: these values are  in the database. They are not hardcoded?